src/rgw/driver/sfs/sqlite/dbconn.h

```diff
@@ -270,7 +272,10 @@ class DBConn {
   DBConn(const DBConn&) = delete;
   DBConn& operator=(const DBConn&) = delete;
 
-  inline auto get_storage() const { return storage; }
+  StorageRef get_storage();
+  sqlite3* get_sqlite_conn() const {
```

Comment:
Does this need     std::shared_lock lock(storage_pool_mutex); ?

Reply:
Yeah, probably.